src/nl/hannahsten/texifyidea/settings/conventions/TexifyConventionsScheme.kt

```diff
@@ -0,0 +1,70 @@
+package nl.hannahsten.texifyidea.settings.conventions
+
+import nl.hannahsten.texifyidea.lang.DefaultEnvironment.*
+import nl.hannahsten.texifyidea.lang.commands.LatexGenericRegularCommand.*
+import nl.hannahsten.texifyidea.lang.commands.LatexListingCommand.LSTINPUTLISTING
+import nl.hannahsten.texifyidea.util.magic.env
+
+/**
+ * A scheme instance stores settings for a specific scope.
+ *
+ * Concrete scheme implementations decide on which scopes make sense. For example, a scheme could represent different
+ * color settings for code or different inspection rules. In the context of Texify conventions, a scheme stores
+ * either project or IDE level conventions.
+ *
+ * Instances must be serializable since they are persisted as part of [TexifyConventionsGlobalState] or
+ * [TexifyConventionsProjectState].
+ */
+data class TexifyConventionsScheme(
+
+    /**
+     * The name of the scheme
+     */
+    var myName: String = DEFAULT_SCHEME_NAME,
+
+    /**
+     * The maximum section size before the corresponding inspection issues a warning.
+     */
+    var maxSectionSize: Int = 4000,
```

Comment:
So this class specifies the default values?

Reply:
Exactly, I added a hint in the documentation.